.github/workflows/mobile-sdk-docs.yml

```
@@ -39,3 +89,47 @@ jobs:
           name: walletsdk-doccarchive
           path: waltid-identity/waltid-libraries/protocols/waltid-wallet-sdk-ios/build/docc/WalletSDK.doccarchive.tar.gz
           if-no-files-found: error
+
+  publish:
```

actionlint (1.7.12)

[error] job "publish" needs job "dokka" which does not exist in this workflow

(job-needs)